homeassistant/components/fully_kiosk/config_flow.py

```diff
@@ -61,3 +64,20 @@ async def async_step_user(
             ),
             errors=errors,
         )
+
+    async def async_step_dhcp(self, discovery_info: DhcpServiceInfo) -> FlowResult:
```

Comment:
For a future PR: Makes me wonder if we can make this a default behavior if DHCP is set to `registered_devices` only ?

Reply:
Depends on what to match... :)

Reply:
We could add that to the manifest too, or just hardcode `CONF_MAC` in `entry.data`. It might be helpful

Reply:
Most integrations set the mac address on their unique ID for the config flow.
Almost all cases just set the unique ID and call an abort with `updates`.